src/main/java/tictactoe/Board.java

```diff
@@ -31,16 +31,16 @@ private int getColumn(int position) {
         return position % size;
     }
 
-    public final MarksEnum[][] getContent() {
+    public MarksEnum[][] getContent() {
```

Comment:
Something to think about is whether you are exposing the underlying data structure of your Board here. Imagine you decided to change the datastructure  to be a List - how much change would you need to make across your code base?  It can be good to create an abstraction so that if the underlying data structure changes, only the Board class needs updating and all other classes remain in tact.